[orgs] Blog post for Organizations launch (#13456)
pypi/warehouse: docs/blog/posts/2023-04-21-introducing-pypi-organizations.md

---
title: Introducing PyPI Organizations
description: Announcing the launch of a signifiant new collaboration feature for PyPI
author: Ee Durbin

> publish_date: 2023-04-21

Author: ```suggestion
publish_date: 2023-04-23
```

packages. We’re making organizations available to community projects for free,
forever, and to corporate projects for a small fee. Priority support will also
be available to all paid subscribers, and all revenue will go right back into
PyPI to continue building better support and infrastructure for all our users.

### Increasing sustainability and support

In 2022 PyPI served 199,529,846,822 downloads for the 428,120 projects hosted
there. This means that in 2022, PyPI saw a 57% growth in download counts and
bandwidth alike. Having more people using and contributing to Python every year
is an fantastic problem to have, but it is one we must increase organizational
capacity to accommodate. Increased revenue for PyPI allows it to become a
staffed platform, one that can respond to support requests and attend to issues
in a timeframe that is significantly faster than what our excellent (but thinly
spread) largely volunteer team could reasonably handle.

### Organizations are opt-in

We also want to be very clear—these new features are completely optional. If
features for larger projects don't sound like something that would be useful to

> you as a PyPI maintainer, then there is no obligation to create a team and

Reviewer: "create an organization" instead of a team?

---
title: Introducing PyPI Organizations
description: Announcing the launch of a signifiant new collaboration feature for PyPI
author: Ee Durbin
publish_date: 2023-04-23
tags:
  - organizations
  - sustainability
---

> Today, we are rolling out the first step in our plan to increase financial

Author: ```suggestion
Today, we are rolling out the first step in our plan to build financial
```

support and long-term sustainability of the Python Packaging Index (PyPI),
while simultaneously giving our users one of our most requested features:
organization accounts.

### Introducing Organizations

Organizations on PyPI are self-managed teams, with their own exclusive branded
web addresses. Our goal is to make PyPI easier to use for large community
projects, organizations, or companies who manage multiple sub-teams and multiple
packages. We’re making organizations available to community projects for free,
forever, and to corporate projects for a small fee. Additional priority support

> agreemens will be available to all paid subscribers, and all revenue will go

Author: ```suggestion
agreements will be available to all paid subscribers, and all revenue will go
```

right back into PyPI to continue building better support and infrastructure
for all our users.

### Increasing sustainability and support

> In 2022 PyPI served 199,529,846,822 downloads for the 428,120 projects hosted

Reviewer 2: ```suggestion
In the last year PyPI served 235.7 billion downloads for the 448,941 projects hosted
```

> there. This means that in 2022, PyPI saw a 57% annual growth in download counts

Reviewer 2: ```suggestion
there. This means that in since the previous period, PyPI saw a 57% annual growth in download counts
```
